lib/rubocop/cop/mixin/nil_methods.rb

```diff
@@ -8,12 +8,16 @@ module NilMethods
       private
 
       def nil_methods
-        nil.methods + whitelist
+        nil.methods + other_stdlib_methods + whitelist
       end
 
       def whitelist
         cop_config['Whitelist'].map(&:to_sym)
       end
+
+      def other_stdlib_methods
+        [:to_d]
```

Comment:
Why just not add this to the default whitelist?

Reply:
(if we keep the current approach we'll also need to update the description of the class)

Reply:
I find keeping the three separate slightly cleaner. Updated class description accordingly. Also rebased the PR.